common/views/themes/config.ts

```diff
@@ -158,6 +158,18 @@ const charcoalWhiteCharcoal: ButtonColors = {
   text: 'charcoal',
 };
 
+const greenTransparentGreen: ButtonColors = {
```

Comment:
Is there a TODO to leave there to address what we discussed this morning about reviewing which variants we want to allow?